### PR TITLE
Homework - UniquePtr

### PR DESCRIPTION
**Pull request by MaciejSalamonski**

### CODE REVIEW

**Review by WBigaj — APPROVED** [review]
I have checked your code and it looks good.

**Review by jakjan95 — APPROVED** [review]
I have no comments :) Good code

File: .github/workflows/UniquePtr.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      working-directory: homework/unique_ptr/
+      run: mkdir build
+    - name: Compile
+      working-directory: homework/unique_ptr/build
+      run: |
+        cmake ..
+        make
+    - name: Run tests
+      working-directory: homework/unique_ptr/build
+      run: ./uniquePtr-ut
```
Comment (niemarcin) [inline .github/workflows/UniquePtr.yml:26]: You can use ctest, as you already enabled testing in cmake file.

**Review by niemarcin — APPROVED** [review]
Add new lines at the end of your files. Github actually marks where you should add it :)

**Review by ziobron — APPROVED** [review]
Very good code 👍

**Comment by ziobron** [comment]
{ "requirement_ids": [3784] }
36 XP za implementację unique_ptra (3784)

**Pull request closed without merging**